apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component, PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import SectionRow, { sectionShape } from './SectionRow';
+
+const styles = {
+  table: {
+    width: '100%',
+    tableLayout: 'auto',
+    backgroundColor: 'transparent',
+    borderSpacing: 0,
+    borderCollapse: 'collapse'
+  },
+  headerRow: {
+    height: 45,
+    paddingTop: 5,
+    paddingBottom: 5,
+    paddingLeft: 15,
+    paddingRight: 15,
+    backgroundColor: color.teal,
+    color: color.white,
+    borderColor: color.white,
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderTopColor: color.teal,
+    borderLeftColor: color.teal,
+    borderRightColor: color.teal,
+    textAlign: 'left'
+  },
+  link: {
+    color: color.white
+  }
+};
+
+export default class SectionTable extends Component {
```

Comment:
Nothing yet. Was planning to do that in sync with adding tests as one of my next steps.